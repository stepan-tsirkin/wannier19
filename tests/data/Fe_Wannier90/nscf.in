 &control
    calculation  = 'bands'
    restart_mode = 'from_scratch'
    pseudo_dir   = '.'
    outdir       = './temp'
    prefix       = 'Fe'
 /
 &system
    ibrav        = 3
    celldm(1)    = 5.4235
    nat          = 1
    ntyp         = 1
    ecutwfc      = 120.d0
    ecutrho      = 600.d0
    lspinorb     = .true.
    noncolin     = .true.
    starting_magnetization(1) = -1.d0
    occupations  = 'smearing'
    smearing     = 'm-v'
    degauss      = 0.02
    nbnd         = 28
 /
 &electrons
    diago_full_acc = .true.
    conv_thr     = 1.0e-10
 /
ATOMIC_SPECIES
 Fe 55.85 Fe.dojo.fr.ONCVv0.4.pbe.standard.upf
ATOMIC_POSITIONS alat
 Fe 0.0 0.0 0.0
K_POINTS crystal
27
  0.00000000  0.00000000  0.00000000  3.703704e-02
  0.00000000  0.00000000  0.33333333  3.703704e-02
  0.00000000  0.00000000  0.66666667  3.703704e-02
  0.00000000  0.33333333  0.00000000  3.703704e-02
  0.00000000  0.33333333  0.33333333  3.703704e-02
  0.00000000  0.33333333  0.66666667  3.703704e-02
  0.00000000  0.66666667  0.00000000  3.703704e-02
  0.00000000  0.66666667  0.33333333  3.703704e-02
  0.00000000  0.66666667  0.66666667  3.703704e-02
  0.33333333  0.00000000  0.00000000  3.703704e-02
  0.33333333  0.00000000  0.33333333  3.703704e-02
  0.33333333  0.00000000  0.66666667  3.703704e-02
  0.33333333  0.33333333  0.00000000  3.703704e-02
  0.33333333  0.33333333  0.33333333  3.703704e-02
  0.33333333  0.33333333  0.66666667  3.703704e-02
  0.33333333  0.66666667  0.00000000  3.703704e-02
  0.33333333  0.66666667  0.33333333  3.703704e-02
  0.33333333  0.66666667  0.66666667  3.703704e-02
  0.66666667  0.00000000  0.00000000  3.703704e-02
  0.66666667  0.00000000  0.33333333  3.703704e-02
  0.66666667  0.00000000  0.66666667  3.703704e-02
  0.66666667  0.33333333  0.00000000  3.703704e-02
  0.66666667  0.33333333  0.33333333  3.703704e-02
  0.66666667  0.33333333  0.66666667  3.703704e-02
  0.66666667  0.66666667  0.00000000  3.703704e-02
  0.66666667  0.66666667  0.33333333  3.703704e-02
  0.66666667  0.66666667  0.66666667  3.703704e-02
