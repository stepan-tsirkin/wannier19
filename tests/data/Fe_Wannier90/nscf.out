
     Program PWSCF v.6.7GPU starts on 22Mar2021 at 13: 3:47 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
         "P. Giannozzi et al., J. Phys.:Condens. Matter 29 465901 (2017);
         "P. Giannozzi et al., J. Chem. Phys. 152 154105 (2020);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Parallel version (MPI & OpenMP), running on      16 processor cores
     Number of MPI processes:                16
     Threads/MPI process:                     1

     MPI processes distributed on     1 nodes
     K-points division:     npool     =      16
     Reading input from nscf.in

     Current dimensions of program PWSCF are:
     Max number of different atomic species (ntypx) = 10
     Max number of k-points (npk) =  40000
     Max angular momentum in pseudopotentials (lmaxx) =  3

     Atomic positions and unit cell read from directory:
     ./temp/Fe.save/
 

     Fixed quantization axis for GGA:     0.000000    0.000000   -1.000000

     Subspace diagonalization in iterative solution of the eigenvalue problem:
     a serial algorithm will be used

 
     G-vector sticks info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Sum        1001     797    277                19861    14171    2899
 
     Using Slab Decomposition
 


     bravais-lattice index     =            3
     lattice parameter (alat)  =       5.4235  a.u.
     unit-cell volume          =      79.7644 (a.u.)^3
     number of atoms/cell      =            1
     number of atomic types    =            1
     number of electrons       =        16.00
     number of Kohn-Sham states=           28
     kinetic-energy cutoff     =     120.0000  Ry
     charge density cutoff     =     600.0000  Ry
     Exchange-correlation= PBE
                           (   1   4   3   4   0   0   0)
     Noncollinear calculation with spin-orbit


     celldm(1)=   5.423500  celldm(2)=   0.000000  celldm(3)=   0.000000
     celldm(4)=   0.000000  celldm(5)=   0.000000  celldm(6)=   0.000000

     crystal axes: (cart. coord. in units of alat)
               a(1) = (   0.500000   0.500000   0.500000 )  
               a(2) = (  -0.500000   0.500000   0.500000 )  
               a(3) = (  -0.500000  -0.500000   0.500000 )  

     reciprocal axes: (cart. coord. in units 2 pi/alat)
               b(1) = (  1.000000  0.000000  1.000000 )  
               b(2) = ( -1.000000  1.000000  0.000000 )  
               b(3) = (  0.000000 -1.000000  1.000000 )  


     PseudoPot. # 1 for Fe read from file:
     ./Fe.dojo.fr.ONCVv0.4.pbe.standard.upf
     MD5 check sum: b9cfc4dc94fead0ebd2746b016929f5e
     Pseudo is Norm-conserving + core correction, Zval = 16.0
     Generated using ONCVPSP code by D. R. Hamann
     Using radial grid of 1426 points, 10 beta functions with: 
                l(1) =   0
                l(2) =   0
                l(3) =   1
                l(4) =   1
                l(5) =   1
                l(6) =   1
                l(7) =   2
                l(8) =   2
                l(9) =   2
               l(10) =   2

     atomic species   valence    mass     pseudopotential
        Fe            16.00    55.85000     Fe( 1.00)

     16 Sym. Ops., with inversion, found



   Cartesian axes

     site n.     atom                  positions (alat units)
         1           Fe  tau(   1) = (   0.0000000   0.0000000   0.0000000  )

     number of k points=    27  Marzari-Vanderbilt smearing, width (Ry)=  0.0200
                       cart. coord. in units 2pi/alat
        k(    1) = (   0.0000000   0.0000000   0.0000000), wk =   0.0370370
        k(    2) = (   0.0000000  -0.3333333   0.3333333), wk =   0.0370370
        k(    3) = (   0.0000000  -0.6666667   0.6666667), wk =   0.0370370
        k(    4) = (  -0.3333333   0.3333333   0.0000000), wk =   0.0370370
        k(    5) = (  -0.3333333   0.0000000   0.3333333), wk =   0.0370370
        k(    6) = (  -0.3333333  -0.3333333   0.6666667), wk =   0.0370370
        k(    7) = (  -0.6666667   0.6666667   0.0000000), wk =   0.0370370
        k(    8) = (  -0.6666667   0.3333333   0.3333333), wk =   0.0370370
        k(    9) = (  -0.6666667   0.0000000   0.6666667), wk =   0.0370370
        k(   10) = (   0.3333333   0.0000000   0.3333333), wk =   0.0370370
        k(   11) = (   0.3333333  -0.3333333   0.6666667), wk =   0.0370370
        k(   12) = (   0.3333333  -0.6666667   1.0000000), wk =   0.0370370
        k(   13) = (   0.0000000   0.3333333   0.3333333), wk =   0.0370370
        k(   14) = (   0.0000000   0.0000000   0.6666667), wk =   0.0370370
        k(   15) = (   0.0000000  -0.3333333   1.0000000), wk =   0.0370370
        k(   16) = (  -0.3333333   0.6666667   0.3333333), wk =   0.0370370
        k(   17) = (  -0.3333333   0.3333333   0.6666667), wk =   0.0370370
        k(   18) = (  -0.3333333   0.0000000   1.0000000), wk =   0.0370370
        k(   19) = (   0.6666667   0.0000000   0.6666667), wk =   0.0370370
        k(   20) = (   0.6666667  -0.3333333   1.0000000), wk =   0.0370370
        k(   21) = (   0.6666667  -0.6666667   1.3333333), wk =   0.0370370
        k(   22) = (   0.3333333   0.3333333   0.6666667), wk =   0.0370370
        k(   23) = (   0.3333333   0.0000000   1.0000000), wk =   0.0370370
        k(   24) = (   0.3333333  -0.3333333   1.3333333), wk =   0.0370370
        k(   25) = (   0.0000000   0.6666667   0.6666667), wk =   0.0370370
        k(   26) = (   0.0000000   0.3333333   1.0000000), wk =   0.0370370
        k(   27) = (   0.0000000   0.0000000   1.3333333), wk =   0.0370370

     Dense  grid:    19861 G-vectors     FFT dimensions: (  40,  40,  40)

     Smooth grid:    14171 G-vectors     FFT dimensions: (  36,  36,  36)

     Estimated max dynamical RAM per process >      29.36 MB

     Estimated total dynamical RAM >     469.77 MB
     Generating pointlists ...
     new r_m :   0.3572 (alat units)  1.9375 (a.u.) for type    1

     The potential is recalculated from file :
     ./temp/Fe.save/charge-density

     Starting wfcs are   20 randomized atomic wfcs +    8 random wfcs

     Band Structure Calculation
     Davidson diagonalization with overlap

     ethr =  6.25E-13,  avg # of iterations = 23.6

     total cpu time spent up to now is        4.9 secs

     End of band structure calculation

          k = 0.0000 0.0000 0.0000 (  1745 PWs)   bands (ev):

   -70.3072 -67.6822 -36.9008 -36.4474 -35.7134 -33.7688 -33.3127 -32.9503
     9.4432   9.4994  15.3190  15.3466  15.3779  16.4846  16.4921  17.5484
    17.5704  17.6016  19.5738  19.5751  41.4714  41.4714  41.4715  41.7707
    41.7707  41.7709  46.9914  46.9917

          k = 0.0000-0.3333 0.3333 (  1762 PWs)   bands (ev):

   -70.2570 -67.6279 -37.0275 -36.5744 -35.8196 -33.9157 -33.4697 -33.0525
    12.5804  13.1518  14.5721  15.7032  16.4737  16.5879  16.6294  17.1339
    17.1974  19.3080  19.6188  19.7341  29.2743  30.1615  36.7414  36.7484
    38.9541  39.5749  40.6564  41.0761

          k = 0.0000-0.6667 0.6667 (  1762 PWs)   bands (ev):

   -70.2570 -67.6279 -37.0275 -36.5744 -35.8196 -33.9157 -33.4697 -33.0525
    12.5804  13.1518  14.5721  15.7032  16.4737  16.5879  16.6294  17.1339
    17.1974  19.3080  19.6188  19.7341  29.2743  30.1615  36.7414  36.7484
    38.9541  39.5749  40.6564  41.0761

          k =-0.3333 0.3333 0.0000 (  1762 PWs)   bands (ev):

   -70.2570 -67.6279 -37.0432 -36.5412 -35.8391 -33.9389 -33.4165 -33.0806
    12.5805  13.1518  14.5722  15.7030  16.4485  16.6139  16.6367  17.1137
    17.2098  19.3059  19.6226  19.7324  29.2744  30.1615  36.7412  36.7488
    38.9537  39.5749  40.6563  41.0762

          k =-0.3333 0.0000 0.3333 (  1762 PWs)   bands (ev):

   -70.2570 -67.6279 -37.0275 -36.5744 -35.8196 -33.9157 -33.4697 -33.0525
    12.5804  13.1518  14.5721  15.7032  16.4737  16.5879  16.6294  17.1339
    17.1974  19.3080  19.6188  19.7341  29.2743  30.1615  36.7414  36.7484
    38.9541  39.5749  40.6564  41.0761

          k =-0.3333-0.3333 0.6667 (  1773 PWs)   bands (ev):

   -70.2318 -67.6007 -37.0844 -36.6252 -35.8906 -33.9803 -33.5256 -33.1339
    14.0537  14.5899  14.6234  15.7981  16.2634  16.2776  16.7608  16.7993
    19.5952  19.6330  24.1931  24.2164  25.3588  25.3608  26.6696  27.5667
    36.1164  36.5850  45.1161  45.1349

          k =-0.6667 0.6667 0.0000 (  1762 PWs)   bands (ev):

   -70.2570 -67.6279 -37.0432 -36.5412 -35.8391 -33.9389 -33.4165 -33.0806
    12.5805  13.1518  14.5722  15.7030  16.4485  16.6139  16.6367  17.1137
    17.2098  19.3059  19.6226  19.7324  29.2744  30.1615  36.7412  36.7488
    38.9537  39.5749  40.6563  41.0762

          k =-0.6667 0.3333 0.3333 (  1773 PWs)   bands (ev):

   -70.2318 -67.6007 -37.0844 -36.6252 -35.8906 -33.9803 -33.5256 -33.1339
    14.0537  14.5899  14.6234  15.7981  16.2634  16.2776  16.7608  16.7993
    19.5952  19.6330  24.1931  24.2164  25.3588  25.3608  26.6696  27.5667
    36.1164  36.5850  45.1161  45.1349

          k =-0.6667 0.0000 0.6667 (  1762 PWs)   bands (ev):

   -70.2570 -67.6279 -37.0275 -36.5744 -35.8196 -33.9157 -33.4697 -33.0525
    12.5804  13.1518  14.5721  15.7032  16.4737  16.5879  16.6294  17.1339
    17.1974  19.3080  19.6188  19.7341  29.2743  30.1615  36.7414  36.7484
    38.9541  39.5749  40.6564  41.0761

          k = 0.3333 0.0000 0.3333 (  1762 PWs)   bands (ev):

   -70.2570 -67.6279 -37.0275 -36.5744 -35.8196 -33.9157 -33.4697 -33.0525
    12.5804  13.1518  14.5721  15.7032  16.4737  16.5879  16.6294  17.1339
    17.1974  19.3080  19.6188  19.7341  29.2743  30.1615  36.7414  36.7484
    38.9541  39.5749  40.6564  41.0761

          k = 0.3333-0.3333 0.6667 (  1773 PWs)   bands (ev):

   -70.2318 -67.6007 -37.0844 -36.6252 -35.8906 -33.9803 -33.5256 -33.1339
    14.0537  14.5899  14.6234  15.7981  16.2634  16.2776  16.7608  16.7993
    19.5952  19.6330  24.1931  24.2164  25.3588  25.3608  26.6696  27.5667
    36.1164  36.5850  45.1161  45.1349

          k = 0.3333-0.6667 1.0000 (  1762 PWs)   bands (ev):

   -70.2570 -67.6279 -37.0432 -36.5412 -35.8391 -33.9389 -33.4165 -33.0806
    12.5805  13.1518  14.5722  15.7030  16.4485  16.6139  16.6367  17.1137
    17.2098  19.3059  19.6226  19.7324  29.2744  30.1615  36.7412  36.7488
    38.9537  39.5749  40.6563  41.0762

          k = 0.0000 0.3333 0.3333 (  1762 PWs)   bands (ev):

   -70.2570 -67.6279 -37.0275 -36.5744 -35.8196 -33.9157 -33.4697 -33.0525
    12.5804  13.1518  14.5721  15.7032  16.4737  16.5879  16.6294  17.1339
    17.1974  19.3080  19.6188  19.7341  29.2743  30.1615  36.7414  36.7484
    38.9541  39.5749  40.6564  41.0761

          k = 0.0000 0.0000 0.6667 (  1778 PWs)   bands (ev):

   -70.2165 -67.5844 -37.0873 -36.6769 -35.9123 -33.9829 -33.5774 -33.1695
    13.5569  13.8218  15.0408  16.0534  16.3915  16.4626  17.0480  18.6014
    18.6723  19.5044  20.3843  21.8593  30.7136  30.8776  31.2607  31.4209
    33.5848  34.4908  35.4864  36.0335

          k = 0.0000-0.3333 1.0000 (  1778 PWs)   bands (ev):

   -70.2165 -67.5844 -37.1062 -36.6384 -35.9349 -34.0024 -33.5290 -33.1954
    13.5569  13.8216  15.0414  16.0521  16.4257  16.4261  17.0509  18.6357
    18.6376  19.5051  20.3838  21.8598  30.7837  30.7839  31.3526  31.3526
    33.5845  34.4906  35.4870  36.0336

          k =-0.3333 0.6667 0.3333 (  1773 PWs)   bands (ev):

   -70.2318 -67.6007 -37.0844 -36.6252 -35.8906 -33.9803 -33.5256 -33.1339
    14.0537  14.5899  14.6234  15.7981  16.2634  16.2776  16.7608  16.7993
    19.5952  19.6330  24.1931  24.2164  25.3588  25.3608  26.6696  27.5667
    36.1164  36.5850  45.1161  45.1349

          k =-0.3333 0.3333 0.6667 (  1773 PWs)   bands (ev):

   -70.2318 -67.6007 -37.0844 -36.6252 -35.8906 -33.9803 -33.5256 -33.1339
    14.0537  14.5899  14.6234  15.7981  16.2634  16.2776  16.7608  16.7993
    19.5952  19.6330  24.1931  24.2164  25.3588  25.3608  26.6696  27.5667
    36.1164  36.5850  45.1161  45.1349

          k =-0.3333 0.0000 1.0000 (  1778 PWs)   bands (ev):

   -70.2165 -67.5844 -37.1062 -36.6384 -35.9349 -34.0024 -33.5290 -33.1954
    13.5569  13.8216  15.0414  16.0521  16.4257  16.4261  17.0509  18.6357
    18.6376  19.5051  20.3838  21.8598  30.7837  30.7839  31.3526  31.3526
    33.5845  34.4906  35.4870  36.0336

          k = 0.6667 0.0000 0.6667 (  1762 PWs)   bands (ev):

   -70.2570 -67.6279 -37.0275 -36.5744 -35.8196 -33.9157 -33.4697 -33.0525
    12.5804  13.1518  14.5721  15.7032  16.4737  16.5879  16.6294  17.1339
    17.1974  19.3080  19.6188  19.7341  29.2743  30.1615  36.7414  36.7484
    38.9541  39.5749  40.6564  41.0761

          k = 0.6667-0.3333 1.0000 (  1762 PWs)   bands (ev):

   -70.2570 -67.6279 -37.0432 -36.5412 -35.8391 -33.9389 -33.4165 -33.0806
    12.5805  13.1518  14.5722  15.7030  16.4485  16.6139  16.6367  17.1137
    17.2098  19.3059  19.6226  19.7324  29.2744  30.1615  36.7412  36.7488
    38.9537  39.5749  40.6563  41.0762

          k = 0.6667-0.6667 1.3333 (  1773 PWs)   bands (ev):

   -70.2318 -67.6007 -37.0844 -36.6252 -35.8906 -33.9803 -33.5256 -33.1339
    14.0537  14.5899  14.6234  15.7981  16.2634  16.2776  16.7608  16.7993
    19.5952  19.6330  24.1931  24.2164  25.3588  25.3608  26.6696  27.5667
    36.1164  36.5850  45.1161  45.1349

          k = 0.3333 0.3333 0.6667 (  1773 PWs)   bands (ev):

   -70.2318 -67.6007 -37.0844 -36.6252 -35.8906 -33.9803 -33.5256 -33.1339
    14.0537  14.5899  14.6234  15.7981  16.2634  16.2776  16.7608  16.7993
    19.5952  19.6330  24.1931  24.2164  25.3588  25.3608  26.6696  27.5667
    36.1164  36.5850  45.1161  45.1349

          k = 0.3333 0.0000 1.0000 (  1778 PWs)   bands (ev):

   -70.2165 -67.5844 -37.1062 -36.6384 -35.9349 -34.0024 -33.5290 -33.1954
    13.5569  13.8216  15.0414  16.0521  16.4257  16.4261  17.0509  18.6357
    18.6376  19.5051  20.3838  21.8598  30.7837  30.7839  31.3526  31.3526
    33.5845  34.4906  35.4870  36.0336

          k = 0.3333-0.3333 1.3333 (  1773 PWs)   bands (ev):

   -70.2318 -67.6007 -37.0844 -36.6252 -35.8906 -33.9803 -33.5256 -33.1339
    14.0537  14.5899  14.6234  15.7981  16.2634  16.2776  16.7608  16.7993
    19.5952  19.6330  24.1931  24.2164  25.3588  25.3608  26.6696  27.5667
    36.1164  36.5850  45.1161  45.1349

          k = 0.0000 0.6667 0.6667 (  1762 PWs)   bands (ev):

   -70.2570 -67.6279 -37.0275 -36.5744 -35.8196 -33.9157 -33.4697 -33.0525
    12.5804  13.1518  14.5721  15.7032  16.4737  16.5879  16.6294  17.1339
    17.1974  19.3080  19.6188  19.7341  29.2743  30.1615  36.7414  36.7484
    38.9541  39.5749  40.6564  41.0761

          k = 0.0000 0.3333 1.0000 (  1778 PWs)   bands (ev):

   -70.2165 -67.5844 -37.1062 -36.6384 -35.9349 -34.0024 -33.5290 -33.1954
    13.5569  13.8216  15.0414  16.0521  16.4257  16.4261  17.0509  18.6357
    18.6376  19.5051  20.3838  21.8598  30.7837  30.7839  31.3526  31.3526
    33.5845  34.4906  35.4870  36.0336

          k = 0.0000 0.0000 1.3333 (  1778 PWs)   bands (ev):

   -70.2165 -67.5844 -37.0873 -36.6769 -35.9123 -33.9829 -33.5774 -33.1695
    13.5569  13.8218  15.0408  16.0534  16.3915  16.4626  17.0480  18.6014
    18.6723  19.5044  20.3843  21.8593  30.7136  30.8776  31.2607  31.4209
    33.5848  34.4908  35.4864  36.0335

     Writing output data file ./temp/Fe.save/
 
     init_run     :      0.28s CPU      0.31s WALL (       1 calls)
     electrons    :      3.73s CPU      3.78s WALL (       1 calls)

     Called by init_run:
     wfcinit      :      0.00s CPU      0.00s WALL (       1 calls)
     potinit      :      0.12s CPU      0.14s WALL (       1 calls)
     hinit0       :      0.14s CPU      0.14s WALL (       1 calls)

     Called by electrons:
     c_bands      :      3.73s CPU      3.78s WALL (       1 calls)
     v_of_rho     :      0.11s CPU      0.11s WALL (       1 calls)

     Called by c_bands:
     init_us_2    :      0.00s CPU      0.00s WALL (       2 calls)
     cegterg      :      3.41s CPU      3.42s WALL (       5 calls)

     Called by *egterg:
     cdiaghg      :      0.08s CPU      0.08s WALL (      70 calls)
     h_psi        :      2.95s CPU      2.96s WALL (      75 calls)
     g_psi        :      0.02s CPU      0.03s WALL (      68 calls)

     Called by h_psi:
     h_psi:calbec :      0.07s CPU      0.07s WALL (      75 calls)
     vloc_psi     :      2.81s CPU      2.82s WALL (      75 calls)
     add_vuspsi   :      0.06s CPU      0.07s WALL (      75 calls)

     General routines
     calbec       :      0.07s CPU      0.07s WALL (      75 calls)
     fft          :      0.03s CPU      0.03s WALL (      27 calls)
     ffts         :      0.00s CPU      0.00s WALL (       4 calls)
     fftw         :      2.31s CPU      2.31s WALL (    4036 calls)
     interpolate  :      0.01s CPU      0.01s WALL (       4 calls)
     davcio       :      0.00s CPU      0.04s WALL (       4 calls)
 
     Parallel routines
 
     PWSCF        :      4.77s CPU      5.61s WALL

 
   This run was terminated on:  13: 3:53  22Mar2021            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
